libs/designsystem/src/lib/components/item/item.component.integration.spec.ts

```diff
@@ -1,177 +1,64 @@
-        spectator.detectChanges();
-        const kirbyItemsInList = spectator.queryAll('kirby-list-item:last-child');
-
-        expect(kirbyItemsInList).not.toBeEmpty();
-        kirbyItemsInList.forEach((item) => {
-          expect(item).toHaveComputedStyle({ 'margin-bottom': '0px' });
-        });
-      });
-    });
-
-    describe('with hasItemSpacing set to false', () => {
-      it('should not apply spacing to items', () => {
-        spectator.setInput('hasItemSpacing', false);
-        spectator.detectChanges();
-        const kirbyItemsInList = spectator.queryAll('kirby-list-item');
-
-        expect(kirbyItemsInList).not.toBeEmpty();
-        kirbyItemsInList.forEach((item) => {
-          expect(item).toHaveComputedStyle({ 'margin-bottom': '0px' });
-        });
-      });
-    });
-  });
-
-  describe('inside kirby-list with cards', () => {
-    beforeEach(async () => {
-      spectator = createHost<ListComponent>(
-        `
-        <kirby-list [items]="[{ name: 'Item1' }, { name: 'Item2' }, { name: 'Item3' }]">
-          <kirby-card *kirbyListItemTemplate="let item">
-            <kirby-item>{{ item.name }}</kirby-item>
-          </kirby-card>
-        </kirby-list>
-        `
-      );
-      ionList = spectator.queryHost('ion-list');
-      await TestHelper.whenReady(ionList);
-      itemsInList = spectator.queryAll('ion-list ion-item');
-    });
-
-    it('should create list wrapper', () => {
-      expect(spectator.component).toBeTruthy();
-    });
-
-    it('should render items in list', () => {
-      expect(itemsInList).not.toBeEmpty();
-      itemsInList.forEach((item) => {
-        expect(item.shadowRoot.hasChildNodes()).toBeTrue();
-      });
-    });
-
-    it('should render first and last item without padding top/bottom', async () => {
-      const firstItem = itemsInList[0].shadowRoot.querySelector('.item-native');
-      const lastItem = itemsInList[itemsInList.length - 1].shadowRoot.querySelector('.item-native');
-      expect(firstItem).toHaveComputedStyle({ 'padding-top': '0px' });
-      expect(lastItem).toHaveComputedStyle({ 'padding-bottom': '0px' });
-    });
-  });
-});
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
+
+import { DesignTokenHelper } from '@kirbydesign/core';
+
+import { IconComponent } from '..';
+import { TestHelper } from '../../testing/test-helper';
+
+import { LabelComponent } from '.';
+import { ItemComponent } from './item.component';
+
+const { fontWeight } = DesignTokenHelper;
+
+describe('ItemComponent', () => {
+  let ionItem: HTMLElement;
+
+  let spectator: SpectatorHost<ItemComponent>;
+  const createHost = createHostFactory({
+    component: ItemComponent,
+    imports: [TestHelper.ionicModuleForTest],
+    declarations: [LabelComponent, IconComponent],
+  });
+
+  describe('selectable and selected with kirby-label', () => {
+    let labelElements: Element[];
+
+    beforeEach(async () => {
+      spectator = createHost<ItemComponent>(
```

Comment:
`createHostFactory` is generic and can be given a type parameter. If it is given `ItemComponent` It will result in spectator having the same type without having to give `createHost` a type parameter every time it is used. 

Therefore consider giving `ItemComponent` as a type parameter to `createHostFactory` on line 17 instead. 
```suggestion
      spectator = createHost(
```